ydb/_topic_reader/topic_reader_sync.py

```diff
@@ -0,0 +1,195 @@
+import asyncio
+import concurrent.futures
+import typing
+from typing import List, Union, Iterable, Optional, Coroutine
+
+from ydb._grpc.grpcwrapper.common_utils import SupportedDriverType
+from ydb._topic_common.common import _get_shared_event_loop
+from ydb._topic_reader.datatypes import PublicMessage, PublicBatch, ICommittable
+from ydb._topic_reader.topic_reader import (
+    PublicReaderSettings,
+    SessionStat,
+    CommitResult,
+)
+from ydb._topic_reader.topic_reader_asyncio import (
+    PublicAsyncIOReader,
+    TopicReaderClosedError,
+)
+
+
+class TopicReaderSync:
+    _loop: asyncio.AbstractEventLoop
+    _async_reader: PublicAsyncIOReader
+    _closed: bool
+
+    def __init__(
+        self,
+        driver: SupportedDriverType,
+        settings: PublicReaderSettings,
+        *,
+        eventloop: Optional[asyncio.AbstractEventLoop] = None,
+    ):
+        self._closed = False
+
+        if eventloop:
+            self._loop = eventloop
+        else:
+            self._loop = _get_shared_event_loop()
+
+        async def create_reader():
+            return PublicAsyncIOReader(driver, settings)
```

Comment:
looks like this don't' need to be async

Reply:
it is ok - it is must be async:
topic reader_sync create async reader and work as wrapper